cypress/integration/login.spec.py.js

```diff
@@ -9,6 +9,31 @@ describe.only('Login', () => {
     loginPage.login('incorrectName', 'incorrectPassword');
     loginPage.expect_Displayed(login.error_message);
   });
+
+  it('logs in when "correct" credentials are entered', () => {
```

Comment:
This test is failing for me.  We may need to mock the request response so it is successful without actual login info

Reply:
Otherwise, looking good! :)

Reply:
Have you entered the proper user name and password into cypress/fixtures/login.json ?
  "user_name": "replace_with_the_correct_user_name",
  "password": "replace_with_the_correct_password",

Reply:
I think we may want to offer a mocked login in the future.
  
For now can you please rename the login.json to login-example.json and update the .gitignore so that it won't ever commit the login.json file.  Then add to the README instructions for developers to create their own login.json with the actual creds in order to run the cypress tests.  Thanks!